tests/test_vqe.py

```diff
@@ -63,6 +63,11 @@ def seed():
     (qml.Hermitian(H_TWO_QUBITS, [0, 1]),),
 ]
 
+OBSERVABLES_NO_HERMITIAN = [
+    (qml.PauliZ(0), qml.PauliY(0), qml.PauliZ(1)),
+    (qml.PauliX(0) @ qml.PauliZ(1), qml.PauliY(0) @ qml.PauliZ(1), qml.PauliZ(1)),
+]
```

Comment:
Since the last testcase  in `OBSERVABLES` is not supported by the new VQE workflow, I made one more test data variable here.